AnnoyancesFilter/sections/popups.txt

```diff
@@ -2188,7 +2190,6 @@ siberkalem.com#$#body { overflow: visible !important; }
 tweaktown.com##.newsletter-overlay
 cyberleninka.ru##body > div.info
 stanleyblackanddecker.com###popup-salesforce-signup
-twitter.com#?##layers > div[class] > div[class] > div[class] > div[class]:has(> div[class] > div[class] > div[class] > div[class] > div[class] > a[href="/login"])
```

Comment:
This rule is still required
![image](https://user-images.githubusercontent.com/8361299/157895197-8554abec-0295-45b1-b0ee-1718c5ea5bca.png)

Reply:
Big popup cannot reproduce.
@AdguardTeam/filters-maintainers Please take a look. Maybe can be blocked by scriptlet(`prevent-addEventListener` on scroll event, or something else)

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/112574#discussion_r824813341
Okay. I pushed.